islandora_base_theme.theme

```diff
@@ -155,37 +155,6 @@ function islandora_base_theme_preprocess_page(&$vars)
  */
 function islandora_base_theme_preprocess_image(&$variables)
 {
-    $variables['iiif_server_location'] = \Drupal::config('islandora_iiif.settings')->get('iiif_server');
-    $variables['file_default_scheme'] = \Drupal::config('system.file')->get('default_scheme');
-    $variables['host'] = \Drupal::request()->getSchemeAndHttpHost();
```

Comment:
Not a user of this theme (nor derivatives of it), but... could there hypothetically be issues downstream of other `hook_preprocess_image()` implementations or templates which expect these variables to exist?

... that said... Drupal's backwards compat policy seems to suggest such being internal:

* https://www.drupal.org/about/core/policies/core-change-policies/drupal-8-and-9-frontend-backwards-compatibility-bc-policy#themes
* https://www.drupal.org/about/core/policies/core-change-policies/drupal-8-and-9-frontend-backwards-compatibility-bc-policy#preprocess-variables
* https://www.drupal.org/about/core/policies/core-change-policies/drupal-8-and-9-backwards-compatibility-and-internal-api#themes
* https://www.drupal.org/about/core/policies/core-change-policies/drupal-8-and-9-backwards-compatibility-and-internal-api#preprocess-variables

... probably fine to drop, if they're indeed not being used.

Reply:
We're currently running a few sites with these commented out and there are no known issues. Thanks for a close read, though!